modules/luci-base/htdocs/luci-static/resources/ui.js

```diff
@@ -3412,6 +3412,20 @@ var UITable = baseclass.extend(/** @lends LuCI.ui.table.prototype */ {
 			if (m)
 				return '%010d%s'.format(+m[1], m[2]);
 
+			m = /^~?((?:\d+)(?:[.]\d+)?)\s(?:(K|M|G)i)?B$/.exec(value);
+
+			if (m)
+				switch (m[2]) {
+					default:
+						return +m[1];
+					case 'K':
```

Comment:
Are you sure about the 'K' here? You seem to mean 'kilo', but 'kilo' is 'k' in short (lower case), not 'K' (upper case).

Reply:
Note the `i` in the regexp: `(?:(K|M|G)i)?B`; it's IEC, not SI.
https://en.wikipedia.org/wiki/Binary_prefix

Reply:
I did miss the 'i' and stand corrected.